### DISCUSSION TITLE
How can I get the latest Observable Framework

### DISCUSSION THREAD

**Discussion by novotny1akub** [observablehq/framework#1074] [Q&A] [answered]
According to the documentation, I initiate the `Observable Framework` using `npm init "@observablehq"`. However, despite following these instructions, I seem to be getting version 1.1.2, as indicated in the Windows command line console. Yet, there are mentions in the documentation of version 1.3. How can I access and utilize this newer version?

**Comment by mbostock** [comment, marked as answer]
You likely need to [clear your npx cache](https://stackoverflow.com/questions/63510325/how-can-i-clear-the-central-cache-for-npx). One way to do that is:

```sh
npx clear-npx-cache
```

**Reply by mbostock** [reply]
You can also specify the latest version explicitly like so:

```sh
npx @observablehq/framework@latest create
```

Or a specific version:

```sh
npx @observablehq/framework@1.3.0 create
```

**Reply by novotny1akub** [reply]
Works like a charm. Thanks, mbostock!
